src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.head-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 1.5rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.container {
+  box-shadow: 0 2px 8px rgb(0 0 0 / 15%);
+  max-width: 50%;
+  margin: 50px auto;
+}
+
+.fa-refresh {
+  color: rgb(119, 124, 129);
+}
+
+.fa-long-arrow-right {
+  color: rgb(119, 124, 129);
+}
+
+.title {
+  padding: 3%;
+  color: rgb(42, 47, 48);
+}
+
+form {
```

Comment:
please always use either `class` or `id` for your tags. If you put the styles property directly to the `tag-name` then it will actually put you in trouble in future in order to make some changes. Like In future if you need to add `media-queries` you would need to add `!important` along with property which is again not considered as Best Practices.